src/audio.rs

```diff
@@ -1,23 +1,28 @@
 //! Loading and playing sounds.
 
+#[cfg(feature = "audio")]
 use crate::{file::load_file, get_context};
-
+#[cfg(feature = "audio")]
 use std::collections::HashMap;
 
 #[cfg(not(target_arch = "wasm32"))]
+#[cfg(feature = "audio")]
 #[path = "audio/native_snd.rs"]
```

Comment:
This is really a lot of #[cfg]! 
Maybe introduce new audio/dummy_snd.rs for use without the audio feature? 
It will behave a little differently - code using audio will still compiles without "audio" feature but will just skip audio output. But maybe this is fine? Especially with way simpler implementation without all the #[cfg]s?